packages/gp2-components/src/molecules/__tests__/UserMenu.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { render } from '@testing-library/react';
+import UserMenu from '../UserMenu';
+
+describe('UserMenu', () => {
+  it('renders the navigation items', () => {
+    const { getAllByRole } = render(<UserMenu />);
```

Comment:
I'd try using screen and not destructuring the result of the render.

Makes it real easy to debug with screen.debug()